lib/renderer/extensions/i18n.js

```diff
@@ -0,0 +1,77 @@
+// Implementation of chrome.i18n.getMessage
+// https://developer.chrome.com/extensions/i18n#method-getMessage
+//
+// Does not implement predefined messages:
+// https://developer.chrome.com/extensions/i18n#overview-predefined
+
+const {ipcRenderer} = require('electron')
+const fs = require('fs')
+const path = require('path')
+
+let metadata
+
+const getExtensionMetadata = () => {
+  if (!metadata) {
+    metadata = ipcRenderer.sendSync('CHROME_I18N_MANIFEST', chrome.runtime.id)
```

Comment:
@zcbenz Any tips for the best way to get the manifest of a devtool's extension from within the extension?

This approach works but not sure if you knew of something cleaner or pre-existing.